### PR TITLE
Add robot_namespace topic publishing to leo_system node

### PR DESCRIPTION
**Pull request by Bitterisland6**
Part of solution for issue in leo_ui repository
Signed-off-by: Bitterisland6 <2010olo@wp.pl>

### CODE REVIEW

File: leo_bringup/scripts/leo_system [hunk 1]
```diff
@@ -16,13 +15,21 @@ def shutdownCallback(x):
     rospy.loginfo("Shutdown command invoked")
     os.system("shutdown -h now")
 
+def publishNamespace():
+    namespace_pub = rospy.Publisher("ros_namespace", String, queue_size=1, latch=True)
+    namespace = os.getenv('ROS_NAMESPACE')
+    msg = "" if namespace == None else str(namespace)
```
Comment (bjsowa) [inline leo_bringup/scripts/leo_system]: You should not rely on the `ROS_NAMESPACE` environment variable. Instead use [rospy.get_namespace()](http://docs.ros.org/en/melodic/api/rospy/html/rospy.names-module.html#get_namespace) method to get the namespace the node was launched in.

**Review by bjsowa — COMMENTED** [review]
Make sure the code follows the [PEP 8](https://www.python.org/dev/peps/pep-0008/) style. You can reformat it using `autopep8` or `black` tools.

File: leo_bringup/scripts/leo_system [hunk 2]
```diff
@@ -17,11 +17,20 @@ def shutdownCallback(x):
     os.system("shutdown -h now")
 
 
+def publishNamespace():
+    namespace_pub = rospy.Publisher("ros_namespace", String, queue_size=1, latch=True)
+    namespace = rospy.get_namespace()
+    msg = "" if namespace == None else str(namespace)
```
Comment (bjsowa) [inline leo_bringup/scripts/leo_system]: Isn't this line redundant when using rospy.get_namespace() ?

Comment (Bitterisland6) [reply leo_bringup/scripts/leo_system]: Yes, it was needed when using os.getenv(). Removing it now.

File: leo_bringup/scripts/leo_system [hunk 3]
```diff
@@ -17,11 +17,19 @@ def shutdownCallback(x):
     os.system("shutdown -h now")
 
 
+def publishNamespace():
+    namespace_pub = rospy.Publisher("ros_namespace", String, queue_size=1, latch=True)
```
Comment (bjsowa) [inline leo_bringup/scripts/leo_system]: I have a feeling `robot_namespace` would be more appropriate name for the topic. This also applies to LeoRover/leo_ui#6

**Review by bjsowa — CHANGES_REQUESTED** [review]

File: leo_bringup/scripts/leo_system (hunk 3 above)
Comment (Bitterisland6) [reply leo_bringup/scripts/leo_system]: Ok, done.

**Review by bjsowa — APPROVED** [review]

**Pull request merged**